helm/superset/templates/_helpers.tpl

```diff
@@ -100,7 +111,11 @@ RESULTS_BACKEND = RedisCache(
       password=env('REDIS_PASSWORD'),
       {{- end }}
       port=env('REDIS_PORT'),
-      key_prefix='superset_results'
+      key_prefix='superset_results',
+      {{- if .Values.supersetNode.connections.use_redis_ssl }}
```

Comment:
`if .Values.supersetNode.connections.redis_ssl`

Reply:
@dpgaspar , fixed it..